contrib/rpcuser/rpcuser.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
```

Comment:
And mention in the readme how this should be called. E.g. `./rpcuser testUserName`.